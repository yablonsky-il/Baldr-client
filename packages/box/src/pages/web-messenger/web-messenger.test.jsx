import WebMessenger from './web-messenger';

describe('<WebMessengerUI />', () => {
  const wrapper = shallow(<WebMessenger />);

  test('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
