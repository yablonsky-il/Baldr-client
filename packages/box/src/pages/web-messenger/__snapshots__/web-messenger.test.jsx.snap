// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WebMessengerUI /> should match snapshot 1`] = `
<WebMessengerCore>
  <Component />
</WebMessengerCore>
`;
